guide/build-config.md

```diff
@@ -234,6 +235,46 @@ You need to specify `group`. Redhat artifacts are then sorted from builds includ
 
 The `MILESTONE` - not implemented yet
 
+The `RESOLVE_ONLY` strategy takes the entry points of the artifacts/extensions of of the product which are supposed to be packaged in repository, resolves the artifact, resolves artifacts managed dependencies. It also takes care of the transitives and package them as well.
+
+ **Necessary configuration**
+ 
+In order to RESOLVE_ONLY strategy to work it requires a link to txt file which contains the artifact or extensions in the format of
+`groupId:artifactId:version:type:classifier` (type and classifier are optional)
+
+*For example*
+
+In build-config.yaml
+
+ ```
+   repositoryGeneration:
+     strategy: RESOLVE_BOM_ONLY
```

Comment:
Please could you change `RESOLVE_BOM_ONLY` to `RESOLVE_ONLY`
I missed it earlier